src/test/java/seedu/address/logic/commands/AddNoteCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_STARTUP;
+import static seedu.address.testutil.TypicalStartups.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.startup.Note;
+import seedu.address.model.startup.Startup;
+import seedu.address.testutil.StartupBuilder;
+
+
+public class AddNoteCommandTest {
```

Comment:
Perhaps we can comment out the failing test cases first so that it can pass the build checks